CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contribution Guideline
+
+## Contribution Welcome!
+
+OpenMLDB is a community-driven opensource project. We welcome all kinds of contributions, such as bugfix, new feature development, test cases, documentations and so on. 
+
+## OpenMLDB Contribution Workflow
+
+### Before Opening Your PR
+
+We encourage you to discuss with us first before coding. Your discussion channel may depend on what you want to contribute:
+
+- If it is a simple bugfix, or an independent function/module that will not affect other modules: You may just raise an issue on our [GitHub Issues](https://github.com/4paradigm/OpenMLDB/issues) and describe your solutions. We will provide necessary feedback to you as soon as possible.
+- If the change may affect a few modules, or an architectural change/improvement: Please contact us in the community and let us have more understanding about your ideas. Meanwhile, we also encourage you to write a formal RFC and post it to the [GitHub Discussions](https://github.com/4paradigm/OpenMLDB/discussions/categories/rfcs).
+
+### Code Contribution Workflow
+
+Here is the general contribution workflow:
+
+- Fork the OpenMLDB repository into your personal space
+- Create a branch from where you want to base your code (usually it is the main branch)
```

Comment:
It's better to link the [code convention](https://github.com/4paradigm/rfcs/blob/main/style-guide/code-convention.md)